app/models/concerns/votes_counter.rb

```diff
@@ -16,7 +16,14 @@ module VotesCounter
   end
 
   def votes_count
-    try(:total_votes) || votes.sum(:value)
+    try(:total_votes) || votes_alternative
+  end
+
+  def votes_alternative
```

Comment:
I think we can remove the `:to_i` and assignment to a variable (`t`). 
```ruby
def votes_alternative
  votes.reduce(0) do |_ , v|
        _ + v
     end
end
```
... and with variable name